configs/setup_golden_image.py

```diff
@@ -46,16 +43,18 @@ def customize_golden(manconfig, golden_tar, build_dir):
         'golden_tar' : golden_tar,
         'build_dir' : build_dir,
         'status_file' : build_dir + '/status.json',
-        'verbose' : VERBOSE,
+        'verbose' : False if logger is None else logger.verbose,
         'debug' : True,         # prevent fork/exec
-        'logger' : None
+        'logger' : logger
     }
 
     response = customize_node.execute(argparse.Namespace(**arg_values))
 
     if response['status'] >= 400:
         msg = '\n !!! -- Customization stage ended with ERROR(s) -- !!!\n%s' %\
                 response['message']
+        if logger is not None:
```

Comment:
This illustrates a basic misunderstanding of logging.  You should never have to do a check like this.  The idea is to always call the Python logging module directly.

Reply:
Hhhmmmm.... I thought that too, but some time ago there was an issue where exception is raise cause "logging is not set" or something like that. Maybe I misunderstood the issue at the time.